test/mongoosePlugin.test.ts

```
@@ -57,17 +56,18 @@ describe('mongoose plugin', () => {
   });
 
   it('returns a promise', () => {
-    const promise = Post.paginate();
+    const promise = (Post as any).paginate();
     expect(promise.then instanceof Function).toBe(true);
   });
 
   it('returns data in the expected format', async () => {
-    const data = await Post.paginate();
+    const data = await (Post as any).paginate();
+    const hasOwnProperty = Object.prototype.hasOwnProperty;
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)